src/test/java/kotlinx/reflect/lite/testData/properties/publicClassValAccessible.kt

```diff
@@ -0,0 +1,14 @@
+package tests.properties.publicClassValAccessible
```

Comment:
This is not stored away in `unusedTestData`, but I don't see it used anywhere. The same is true for *many* other files.

Reply:
Fixed this and also added most of the tests for `KCallable<*>.isAccessible` to `testData`